sdl-core/src/main/scala/io/smartdatalake/util/misc/SchemaUtil.scala

```diff
@@ -87,18 +97,18 @@ object SchemaUtil {
    * @param ignoreNullability whether to ignore differences in nullability.
    * @return `true` iff `leftType` is a subset of `rightType`. `false` otherwise.
    */
-  private def deepIsTypeSubset(leftType: GenericDataType, rightType: GenericDataType, ignoreNullability: Boolean, caseSensitive: Boolean ): Boolean = {
-    if (leftType.typeName != rightType.typeName) false  /*fail fast*/
+  private def deepIsTypeSubset(leftType: GenericDataType, rightType: GenericDataType, ignoreNullability: Boolean, caseSensitive: Boolean): Boolean = {
```

Comment:
In `schemaDiff`, the parameter is called `ignoreNullable`. Can you please use the same term here?

Reply:
done